packages/realm/src/Realm.ts

```diff
@@ -220,14 +223,17 @@ export class Realm {
   public static ClientResetMode = ClientResetMode;
   /** @deprecated Please use named imports */
   public static Collection = Collection;
+  public static CompensatingWriteError = CompensatingWriteError;
```

Comment:
Seems there have been more things added.  Is there a rebase missing?

Reply:
The tests are already accessing this via named imports, but since we're now simply exporting `module.exports = Realm` they need to be exposed as statics off the `Realm` class instead.

Reply:
Ok, thanks for the explanation